README.md

```diff
@@ -60,7 +61,7 @@ suppose('npm', ['init'])
 })
 ```
 
-Always follow an `.on()` with a `.respond()` and then finish with a `.end()`.
+Always follow an `.when()` with a `.respond()` and then finish with a `.end()`.
```

Comment:
That's not true anymore, you can interleave how much `.when()` and `.respond()` as you want.